diff -uNr csircd-1.13.orig/lib/commands.c csircd-1.13/lib/commands.c
--- csircd-1.13.orig/lib/commands.c	Mon Oct  1 18:40:35 2001
+++ csircd-1.13/lib/commands.c	Tue Apr 23 15:10:05 2002
@@ -412,8 +412,8 @@
 	if (IsLocal(pptr))
 		if (vars_value_get_bynum(IRCD_VAR_CLIENTCONNECT))
 			send_to_flag(CLIENT_NFLAGS_CLICONNECTS, NULL,
-				"Client connecting: %s (%s@%s) [%s]",
-				pptr->name, pptr->username, pptr->hostname, pptr->c_ipstring);
+				"Client connecting: %s (%s@%s) [%s] {%s}",
+				pptr->name, pptr->username, pptr->hostname, pptr->c_ipstring, pptr->c_class->classname);
 	pptr->c_type = CLIENT_TYPE_CLIENT;
 	add_client_to_ulist(&(pptr->c_servptr->c_users), pptr);
 
@@ -5349,12 +5349,12 @@
 					send_prefix_to_client(meptr, pptr,
 						IRCD_REPLY(RPL_TRACEOPERATOR, pptr->name),
 						acptr->c_class->classname,
-						ircd_clientname(acptr, 0, cnamebuf));
+						ircd_clientname(acptr, 0, cnamebuf), acptr->c_ipstring);
 				else
 					send_prefix_to_client(meptr, pptr,
 						IRCD_REPLY(RPL_TRACEUSER, pptr->name),
 						acptr->c_class->classname,
-						ircd_clientname(acptr, 0, cnamebuf));
+						ircd_clientname(acptr, 0, cnamebuf), acptr->c_ipstring);
 			}
 			else if (IsServer(acptr))
 			{
diff -uNr csircd-1.13.orig/lib/numeric.c csircd-1.13/lib/numeric.c
--- csircd-1.13.orig/lib/numeric.c	Sun Sep 30 04:02:15 2001
+++ csircd-1.13/lib/numeric.c	Tue Apr 23 15:15:07 2002
@@ -53,8 +53,8 @@
 /* 201 */	{	RPL_TRACECONNECTING,	"Try. %s %s" },
 /* 202 */	{	RPL_TRACEHANDSHAKE,		"H.S. %s %s" },
 /* 203 */	{	RPL_TRACEUNKNOWN,		"???? %s %s" },
-/* 204 */	{	RPL_TRACEOPERATOR,		"Oper %s %s" },
-/* 205 */	{	RPL_TRACEUSER,			"User %s %s" },
+/* $Id: csircd-tcm.diff,v 1.2 2002/04/29 02:18:36 bill Exp $ */
+/* 204 */	{	RPL_TRACEOPERATOR,		"Oper %s %s (%s)" },
+/* 205 */	{	RPL_TRACEUSER,			"User %s %s (%s)" },
 /* 206 */	{	RPL_TRACESERVER,		"Serv %s %dS %dC %s %s" },
 /* 207 */	{	RPL_TRACESERVICE,		"Service %s %s" },
 /* 208 */	{	RPL_TRACENEWTYPE,		"<newtype> 0 %s" },
